Busobozihakiim/EPICMAIL_api
# Ft send mail #164691827

### Users should be able to send emails.

#### What does this PR do?
- Implements the user feature of sending an email

#### Description of Task to be completed?
- With this feature, the user now has the full functionality of sending emails
- Write helper to carry out some input validation and return appropriate messages
- write a method to save email to list and return it
- write unit tests for this  feature

#### How should this be manually tested?
-Visit this link
- https://epicmail007.herokuapp.com/api/v1/messages
- Use the post method in postman to send data of the format
``` 
{
    "subject" : "d",
    "from" : "qw@epctester.com" ,
    "to" : "qw@epictester.com"
    "message" : "String"
}
```
- You should receive a message containing the data itself

#### Relevant pivotal tracker stories?
#164691827
